public/v4/apps/bugsink.yaml

```diff
@@ -0,0 +1,79 @@
+captainVersion: 4
+services:
+    $$cap_appname-mysql:
+        image: mysql:$$cap_MYSQL_VERSION
+        restart: unless-stopped
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_MYSQL_PASSWORD
+            MYSQL_DATABASE: bugsink
+        volumes:
+            - $$cap_appname-mysql-data:/var/lib/mysql
+        healthcheck:
+            test: ["CMD-SHELL", "exit | mysql -h localhost -P 3306 -u root -p$$MYSQL_ROOT_PASSWORD" ]
+            interval: 1s
+            timeout: 20s
+            retries: 30
+
+    $$cap_appname:
+        image: bugsink/bugsink:$$cap_BUGSINK_VERSION
+        depends_on:
+            mysql:
+                condition: service_healthy
+        restart: unless-stopped
+        ports:
+            - "8000:8000"
+        environment:
+            SECRET_KEY: $$cap_gen_random_hex(50)
+            CREATE_SUPERUSER: admin:$$cap_SUPERUSER_PASSWORD
+            PORT: 8000
+            DATABASE_URL: mysql://root:$$cap_MYSQL_PASSWORD@srv-captain--$$cap_appname-mysql:3306/bugsink
+            BASE_URL: https://$$cap_appname.$$cap_root_domain/
+            BEHIND_HTTPS_PROXY: "true"
+        healthcheck:
+            test: ["CMD-SHELL", "python -c 'import requests; requests.get(\"http://localhost:8000/\").raise_for_status()'"]
+            interval: 5s
+            timeout: 20s
+            retries: 10
+        caproverExtra:
+            containerHttpPort: '8000'
+
+volumes:
+    my-datavolume:
+
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_BUGSINK_VERSION'
+          label: Bugsink Version
+          defaultValue: '1.4.2'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/bugsink/bugsink/tags
+          validRegex: '/.{1,}/'
+        - id: $$cap_MYSQL_VERSION
+          label: Database | Version
+          description: MySQL version. Check out their valid tags at https://hub.docker.com/_/mysql/tags
+          defaultValue: '8.4.4'
+          validRegex: /.{1,}/
+        - id: '$$cap_MYSQL_PASSWORD'
+          label: MySQL Password
+          defaultValue: $$cap_gen_random_hex(16)
+          description: The password for the MySQL root user.
+          validRegex: '/.{1,}/'
+        - id: '$$cap_SUPERUSER_PASSWORD'
```

Comment:
Recommend using 
```
defaultValue: $$cap_gen_random_hex(16)
```
for easier installation

Reply:
Same for MySQL password

Reply:
> `defaultValue: $$cap_gen_random_hex(16)`

that's what I wrote, isn't it?

for the superuser password, I intentionally left it blank to avoid the user setting something without noticing it, and then not being able to log in. But if that's not the way to go on caprover, I'll gladly change.